### PR TITLE
refactor: change structure DetailsScreen

### PR DESCRIPTION
**Pull request by koverchik**

### CODE REVIEW

File: src/Screen/DetailsScreen/DetailsScreenView.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+              {t('detailsScreen:code')}: {props.airportData?.country.code}
+            </Text>
+            <Text style={Styles.text}>IATA: {props.airportData?.iata}</Text>
+          </View>
+        </View>
+      </View>
+      <Text style={Styles.headerLink}>{t('detailsScreen:links')}:</Text>
+      <View style={Styles.wrapperLinks}>
+        <FlatList
+          data={props.linksAboutAirport()}
```
Comment (evgenyshenets91) [inline src/Screen/DetailsScreen/DetailsScreenView.tsx:50]: please destruct linksAboutAirport and other props above the line
`const Styles = useThemeAwareObject(createStyles);`

File: src/Screen/DetailsScreen/index.tsx [hunk 2]
```diff
@@ -1,42 +1,51 @@
...
+  const renderItem: ListRenderItem<ItemLinks> = ({ item }) => {
+    return (
+      <TouchableOpacity onPress={() => Linking.openURL(item.link)}>
+        <Text style={Styles.link}>{item.name}</Text>
+      </TouchableOpacity>
+    );
+  };
+
+  const linksAboutAirport = () => {
+    const list: ItemLinks[] = [];
```
Comment (evgenyshenets91) [inline src/Screen/DetailsScreen/index.tsx:41]: I am not sure that we have to add type here. it is an empty array and next you are pushing the correct object with a defined type. I think as result we could see in list ItemLinks[], but let's check it

File: src/Screen/DetailsScreen/index.tsx [hunk 3]
```diff
@@ -46,44 +55,10 @@ export const DetailsScreen: FC<DetailsScreenProps> = (props) => {
...
-      <FlatList
-          data={linksAirports}
-          renderItem={renderItem}
-          keyExtractor={(item) => item.name}
-      />
-      </View>
-    </View>
+    <DetailsScreenView
+      airportData={airportData}
+      linksAboutAirport={linksAboutAirport}
```
Comment (evgenyshenets91) [inline src/Screen/DetailsScreen/index.tsx:60]: please pass here the result of calling the function

File: src/Screen/DetailsScreen/index.tsx [hunk 4]
```diff
@@ -1,42 +1,51 @@
...
-  const [linksAirports, setLinksAirports] = useState<itemLinks[]>([]);
   const { pending, airportData } = useAppSelector(getDetailsAirport);
 
   useEffect(() => {
     dispatch(requestAirportDetails(props.route.params.iata));
   }, []);
 
-  useEffect(() => {
+  const renderItem: ListRenderItem<ItemLinks> = ({ item }) => {
+    return (
```
Comment (evgenyshenets91) [inline src/Screen/DetailsScreen/index.tsx:33]: destruct name and link from item

File: src/Screen/DetailsScreen/index.tsx (hunk 2 above)
Comment (koverchik) [reply src/Screen/DetailsScreen/index.tsx:41]: I tried to delete this type declaration, but ts compilator said me about the problem "TS7034: Variable 'list' implicitly has type 'any[]' in some locations where its type cannot be determined"

**Pull request merged**